src/java/org/apache/cassandra/service/accord/api/AccordAgent.java

```diff
@@ -49,4 +50,10 @@ public void onHandledException(Throwable throwable)
     {
         // TODO: this
     }
+
+    @Override
+    public boolean isExpired(TxnId initiated, long now)
+    {
+        return false;
```

Comment:
No check at all? This is TBD until it can be configured?

Reply:
Yeah, I think we also want to discriminate between write and read transactions here. I don't mind plumbing it in for now, but it's probably better to leave it until later - it's also not essential for correctness, it only applies to long running original coordinators. I vacillated over whether we should keep it at all, but decided it was probably better.

Reply:
As it happens, it looks like the whole Agent is to-be-implemented anyway